src/components/CommandBar/CommandBarKclInput.tsx

```diff
@@ -221,16 +245,16 @@ function CommandBarKclInput({
             spellCheck="false"
             autoFocus
             onChange={(e) => setNewVariableName(e.target.value)}
-            onKeyDown={(e) => {
-              if (e.currentTarget.value === '' && e.key === 'Backspace') {
-                setCreateNewVariable(false)
-              }
-            }}
-            onKeyUp={(e) => {
-              if (e.key === 'Enter') {
-                handleSubmit()
-              }
-            }}
+            // onKeyDown={(e) => {
+            //   if (e.currentTarget.value === '' && e.key === 'Backspace') {
+            //     setCreateNewVariable(false)
+            //   }
+            // }}
+            // onKeyUp={(e) => {
+            //   // if (e.key === 'Enter') {
+            //   //   handleSubmit()
+            //   // }
+            // }}
```

Comment:
[not a blocker] Did you mean to keep those lines?

Reply:
No I did not, let me remove them, thank you